src/packer.pkr.hcl

```diff
@@ -136,19 +137,191 @@ build {
   }
 
   provisioner "windows-restart" {
-    # Wait a maximum of 30 minutes for Windows to restart.
-    # The build will fail if the restart process takes longer than 30 minutes.
-    restart_timeout = "30m"
+    # Wait a maximum of 5 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 5 minutes.
+    restart_timeout = "5m"
   }
 
   provisioner "powershell" {
-    # Wait 90 seconds before executing the check-defender.ps1 powershell script.
-    # This gives a generous grace period between restarting Windows and running the second provisioner.
-    pause_before = "90s"
-    scripts      = ["src/powershell/check-defender.ps1"]
+    # Wait 30 seconds before executing the next provisioner.
+    # This gives a grace period between restarting Windows and running the next provisioner.
+    pause_before = "30s"
+    scripts = [
+      "src/powershell/check-defender.ps1",
+      "src/powershell/enable-rdp.ps1",
+      "src/powershell/initialize-volume.ps1",
+      "src/powershell/install-chocolatey.ps1"
+    ]
   }
 
   provisioner "powershell" {
-    scripts = ["src/powershell/enable-rdp.ps1"]
+    # Create "packages" directory before uploading them in the next provisioner.
+    inline = ["mkdir D:\\packages"]
```

Comment:
I'd like to see a comment here explaining why the D drive is being used.

Reply:
@itsmostafa I like what you did with https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/2d381b7d3a5bad7561e48f818a5191fc19ee279a, but there still should be some explanatory comment about why we are installing everything on the D drive, rather than the C drive.

Reply:
@dav3r resolved with [d8e1587](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/d8e1587a6ea7fc1036afd9e594ac3531b7ebfd82)

Reply:
Great, thank you!